### ISSUE TITLE
Exceptionally low performance in Flametail (DSi title)

### ISSUE THREAD

**Issue by boomercore** [melonDS-emu/melonDS#1878]
All other DSi titles are running well above full framerate unlocked on my 5800X, something seems to be up with this title in particular.  OGL, Software, JIT on or off, no matter what it runs at ~19fps.

**Comment by CasualPokePlayer** [comment]
The game is likely using the DSP which heavily tanks performance.

**Comment by boomercore** [comment]
Makes sense considering the dev blog mentioning how terribly the sound app runs.

**Comment by dxgldotorg** [comment]
The slowdown is still present in 1.0-RC.  Getting 21fps on a degraded to 5.5GHz 13900K.

**Comment by CasualPokePlayer** [comment]
There won't be any fix for this in 1.0.

**Comment by boomercore** [comment]
DSP up for optimization any time soon or is this several rows down the priority list?

**Comment by CasualPokePlayer** [comment]
Optimization of the DSP frankly wouldn't even be the first on the list, having it be functional at for most games would be better (for many games, the DSP seemingly crashes or gets stuck on initialization). Optimization of the DSP too would likely just mean shoving a JIT in or HLE'ing the DSP (I suppose though if HLE DSP is done then it would bypass functionality issues).

**Comment by boomercore** [comment]
Guess I'll play this game when I play it.  Went down a rabbit hole of Mindware's prior works and bumped into this.

**Comment by Arisotura** [comment]
fixed with dsp_hle merge
